_episodes/starting-jekyll.md

```diff
@@ -165,6 +167,84 @@ Let's make use of global parameters in our pages.
 > a single place (even if you only use them once). In combination with Jekyll layouts/templates (to be covered in the next episode) they are a great way of creating reusable markup snippets that can be repeated on multiple or even on every page of your website. Reuse helps you reduce the amount of code you have to write.
 {: .callout}
 
+## When things go wrong
+
+So far we have seen how to successfully use Jekyll to produce a website.
+There are however some situations where Jekyll may fail to do so
+either due to a typo or missing information.
+
+> ## Exercise: Troubleshooting Jekyll
+>
+> This exercise will help you recognise how common mistakes look like
+> when working with these elements of a Jekyll website.
+>
+> Edit your `_config.yml` file and omit a closing quote `"` in one of the variables.
+>
+> > ## Solution
+> >
+> > For instance, a missing closing quote `"` in the `twitter` variable.
+> > ~~~
+> > title: "Building Websites in GitHub"
+> > description: "This research project develops training materials for reseachers wanting to learn to build project
+> > websites in GitHub with GitHub Pages."
+> > email: "team@carpentries.org"
+> > twitter: "https://twitter.com/thecarpentries
+> > ~~~
+> > {: .language-yaml}
+> >
+> > If you navigate your GitHub repository you would be able to see something break in `about.md` where we use `{% raw %}{{ site.twitter }}{% endraw %}` however,
+> > contrary to what we saw before with invalid Markdown,
+> > Jekyll will refuse to build the website and produce an error message.
+> >
+> > We will see after this where to find the error message and identify what caused them.
+> {: .solution }
+{: .challenge }
+
+If you were keeping an eye on the GitHub repository page until now, you may have noticed 
+a yellow circle visible when the website is still being processed and a green check mark (✓) when successful.
+You may have also noticed that in the same location there is now a red cross next to the commit message (❌).
+
+This indicates that something went wrong with our Jekyll build process.
+
+![Jekyll pending/successful build](../fig/includes_fail_pending_successful.png)
+
+You may also find an email from GitHub in your inbox with details about the error.
+But lets look at our repository again.
```

Comment:
```suggestion
But let's look at our repository again.
```

Reply:
I do this one all the time. Thanks Toby!